WcaOnRails/app/views/delegates/probations.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= react_component("DelegateProbations", {
+    probationRoles: @probation_roles,
+    probationUsers: @probation_users,
```

Comment:
In the long run, React components should be responsible for actively fetching data from Rails, instead of Rails passing the data.

Rails should really only pass a minimum amount of information to React. For example, the React component for showing a competition's scrambles receives _only_ the competition ID and then during runtime on client-side starts a request to the Rails backend to fetch scrambles based on the competition ID. It should not receive the scrambles directly during render.

Similarly, your React component should fetch the probation Delegates from the backend (probably using our custom data loading hooks). I'll accept the PR in its current state if a rewrite is too much work, but I still thought I'd let you know :)